### ISSUE TITLE
QUESTION. Using the code to render schematics on a website

### ISSUE THREAD

**Issue by tomasz-brak** [misode/vscode-nbt#60]
Hi, 
appreciate your work, i wonder if there is an easy way to use the renderer you wrote to render schematics on a website. I know it probably is a lot of work but is it even viable? Or would it be easier to start from scratch. I don't know typescript so the code looks like dark magic to me. But i want to create a website with schematics to download and this is the only renderer I could find that didn't suck.

TL;dr Is there a bundled version of the 3d viewer somewhere so it can be used with any HTML document? 
Thanks for help

**Comment by misode** [comment]
@tomasz-brak Have a look at my [deepslate](https://github.com/misode/deepslate) repository, which contains the actual logic for rendering structures. More specifically there is an example in the `demo` folder. It is TypeScript but it should be easy to remove the types if you don't want those. Let me know if you need any more guidance!
